spec/unit/settings_spec.rb

```diff
@@ -129,6 +129,18 @@
         expect { subject.evaluate('puppet_path' => '/nonexistent') }.to raise_error(R10K::Settings::Collection::ValidationError)
       end
     end
+
+    describe 'puppet_conf' do
+      it 'when file raises no error' do
+        expect(File).to receive(:exist?).with('/nonexistent').and_return(true)
+        expect { subject.evaluate('puppet_conf' => '/nonexistent') }.not_to raise_error
+      end
+
+      it 'when not file raises error' do
+        expect(File).to receive(:exist?).with('/nonexistent')
```

Comment:
Would it be worth explicitly returning `false` here?

Reply:
This was fixed as part of changing to readable? check.